tests/dm_syncer/conf/diff_config_route_rules.toml

```diff
@@ -0,0 +1,60 @@
+# diff Configuration.
+
+log-level = "info"
+
+chunk-size = 1000
+
+check-thread-count = 4
+
+sample-percent = 100
+
+use-rowid = false
+
+use-checksum = true
+
+fix-sql-file = "fix.sql"
+
+# tables need to check.
+[[check-tables]]
+schema = "dm_syncer"
+tables = ["dm_syncer_route_rule"]
+
+[[table-config]]
+schema = "dm_syncer"
+table = "dm_syncer_route_rule_1"
```

Comment:
please follow https://docs.pingcap.com/zh/tidb/stable/shard-diff

`table` here should be downstream "merged" table, and should set `is-sharding = true`